2.4-taptree.ipynb

```diff
@@ -543,7 +451,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### _Programming Exercise 2.4.7:_ Sign the transaction for `TapLeafA` \n",
+    "#### _Programming Exercise 2.4.8:_ Sign the transaction for `TapLeafA` \n",
     "\n",
     "Note that we must pass the following arguments to `TaprootSignatureHash` for script path spending:\n",
     "* `scriptpath`: `True`\n",
```

Comment:
[output] should be [tx.vout[0]] based on the solution

_NOTE: I cannot leave feedback on certain lines based on github restrictions. Go down a few lines._

Reply:
fixed